db/init.go

```diff
@@ -104,12 +107,35 @@ func Migrate() error {
 	}
 	defer db.Close()
 
-	if err := goose.Up(db, "migrations", goose.WithAllowMissing()); err != nil {
-		return err
+	for {
+		err = goose.UpByOne(db, "migrations", goose.WithAllowMissing())
+		if err == goose.ErrNoNextVersion {
+			break
+		}
+		if err != nil {
+			return err
+		}
 	}
 	return nil
 }
 
 func GetDB() (*sql.DB, error) {
 	return sql.Open("pgx", pgxConnectionString)
 }
+
+func ConvertNamedParams(sql string, namedArgs map[string]interface{}) (string, []interface{}) {
+	i := 1
+	var args []interface{}
+	// Loop the named args and replace with placeholders
+	for pname, pval := range namedArgs {
+		sql = strings.ReplaceAll(sql, ":"+pname, fmt.Sprint(`$`, i))
+		args = append(args, pval)
+		i++
+	}
```

Comment:
Is it possible to use `sql.Named` here ? It works something like this 
```golang
db.ExecContext(ctx, `
    delete from Invoice
    where
        TimeCreated < @end
        and TimeCreated >= @start;`,
    sql.Named("start", startTime),
    sql.Named("end", endTime),
)
```

Reply:
We don't need this functions since went ahead with the chaining